contrib/caldav/calcurse-caldav.py

```diff
@@ -182,7 +260,12 @@ def remote_wipe(conn):
     if dry_run:
         return
 
-    remote_query(conn, "DELETE", path, {}, None)
+    if authmethod == 'basic':
+        remote_query(conn, "DELETE", path, {}, None)
+    elif authmethod == 'oauth2':
+        remote_items = get_etags(conn)
+        for href in remote_items:
+            remove_remote_object(conn, remote_items[href], href)
```

Comment:
This surprised me quite a bit. You do we need special handling of OAuth2 here?

Reply:
While the original method of remote wiping is significantly faster, it is unfortunately not supported by Google Calendar. Rather than sending a single DELETE request to the calendar collection root, it requires that each event be deleted individually. It is probably something related to security/best practices.

I can add a comment here to document that.

Reply:
Sounds like something which is not related to OAuth2 then. So we should probably:

1. Switch to making separate requests for both basic authentication and OAuth2.
2. Do it in a separate patch.

Reply:
Ok, I will do that.